### ISSUE TITLE
Button activeOpacity prop ignored

### ISSUE THREAD

**Issue by Amurmurmur** [react-native-training/react-native-elements#1190]
### Description

When setting activeOpacity prop to a button, it doesnt get passed into the TouchableOpacity component inside the Button component.

I dont use my custom TouchableOpacity component.

Here is my button code:
```
<Button 
activeOpacity={.8}
containerStyle={styles.containerStyle} 
buttonStyle={styles.buttonStyle} 
titleStyle={styles.titleStyle} 
/>

```

### Possible Fix

I believe this https://github.com/react-native-training/react-native-elements/blob/master/src/buttons/Button.js#L78 has to be spread after https://github.com/react-native-training/react-native-elements/blob/master/src/buttons/Button.js#L82

So that the user can override touchable component attributes

**Comment by iRoachie** [comment]
Can I ask which version of react-native-elements you are using?

**Comment by Amurmurmur** [comment]
@iRoachie   "react-native-elements": "1.0.0-beta5"

**Comment by Amurmurmur** [comment]
@iRoachie Ive created this pull requiest https://github.com/react-native-training/react-native-elements/pull/1191

**Comment by iRoachie** [comment]
Thanks again @Amurmurmur, just merged in. Will be available next release.

**Comment by iRoachie** [comment]
Fixed now in [1.0.0-beta6](https://github.com/react-native-training/react-native-elements/releases/tag/v1.0.0-beta6)!
